src/frontend/ExtensionManager/index.tsx

```diff
@@ -26,27 +27,48 @@ const ExtensionManager = function () {
   const rootRef = useRef<HTMLDivElement>(null)
   const trueAsStr = 'false' as unknown as boolean | undefined
 
+  const mmContainerStyle = {} as React.CSSProperties
+
+  if (OverlayState.title === 'HyperPlay Extension') {
```

Comment:
Maybe we can have a `OverlayState.mode`? We could also have the title derived from the mode. We're doing a string comparison here that looks a bit weird

```ts
type OverlayMode = 'extension' | 'browser-game' | 'web-game';

class OverlayState {
    mode: OverlayMode;

    get title(): {
        const titles: Record<OverlayMode, string> = {
            extension: 'HyperPlay Extension Overlay',
            'browser-game': 'HyperPlay Browser Game',
            'web-game': 'HyperPlay Web Game'
        }
        return titles[this.mode];
    }
}
```